files/image_config/secureboot/whitelist_paths.conf

```diff
@@ -0,0 +1,28 @@
+home/.*
+var/core/.*
+var/log/.*
+etc/group
+etc/gshadow
+etc/hostname
+etc/hosts
+etc/machine-id
+etc/network/interfaces
+etc/nsswitch.conf
+etc/pam.d/common-auth-sonic
+etc/pam.d/sshd
+etc/pam.d/login
+etc/passwd
+etc/rsyslog.conf
+etc/shadow
+etc/sonic/acl.json
+etc/sonic/config_db.json
+etc/sonic/minigraph.xml
+etc/sonic/snmp.yml
+etc/sonic/updategraph.conf
+etc/ssh/ssh_host_rsa_key.pub
+etc/ssh/ssh_host_rsa_key
+etc/subgid
+etc/subuid
+etc/tacplus_nss.conf
+etc/tacplus_user
+lib/systemd/system/serial-getty@.service
```

Comment:
I think you are missing a few files, you can take a look at /host/<image>/rw/ and see which files are changed. Following is a list of files changed under /etc:

admin@str-7260cx3-acs-1:/host/image-20191130.35/rw$ sudo find etc -type f
etc/dhcp/dhclient.conf
etc/hostname
etc/pam.d/sshd
etc/pam.d/login.old
etc/pam.d/common-auth-sonic
etc/pam.d/login
etc/pam.d/sshd.old
etc/machine-id
etc/default/ntp
etc/rsyslog.conf
etc/adjtime
etc/sysctl.d/90-dhcp6-systcl.conf
etc/sonic/minigraph.xml
etc/sonic/frr/frr.conf
etc/sonic/frr/vtysh.conf
etc/sonic/old_config/init_cfg.json
etc/sonic/old_config/asic_config_checksum
etc/sonic/old_config/minigraph.xml
etc/sonic/old_config/frr/frr.conf
etc/sonic/old_config/frr/vtysh.conf
etc/sonic/old_config/deployment_id_asn_map.yml
etc/sonic/old_config/updategraph.conf
etc/sonic/old_config/core_analyzer.rc.json
etc/sonic/old_config/.syseeprom
etc/sonic/old_config/sonic_version.yml
etc/sonic/old_config/config_db.json
etc/sonic/old_config/snmp.yml
etc/sonic/updategraph.conf
etc/sonic/.syseeprom
etc/sonic/config_db.json
etc/sonic/snmp.yml
etc/apparmor.d/cache/.features
etc/apparmor.d/cache/usr.sbin.ntpd
etc/nsswitch.conf
etc/cron.d/logrotate
etc/network/interfaces
etc/hosts.old
etc/nsswitch.conf.old
etc/hosts
etc/ebtables.filter
etc/ssh/ssh_host_rsa_key
etc/ssh/sshd_config
etc/ssh/ssh_host_rsa_key.pub
etc/tacplus_nss.conf
etc/ntp.conf
etc/environment

Reply:
Can we remove the following files?
etc/sonic/old_config/***
etc/nsswitch.conf.old
etc/pam.d/sshd.old

Reply:
Some of the files not existing in prod, no added.